packages/hathor-rpc-handler/__tests__/rpcMethods/sendTransaction.test.ts

```diff
@@ -61,6 +61,11 @@ describe('sendTransaction', () => {
             address: 'testAddress',
             token: '00',
           }],
+          outputs: [{
+            address: 'testAddress',
+            value: BigInt(100),
```

Comment:
Shouldn't all values be bigint? Should we change the input value as well?

Reply:
Yes we should, done